css/styles.css

```diff
@@ -13,13 +13,14 @@ body {
 nav a {
```

Comment:
Let's use SASS. http://sass-lang.com/documentation/file.SASS_REFERENCE.html

I will be the same at the end since the .scss gets converted to .css.